contracts/facets/EmbalmerFacet.sol

```diff
@@ -142,23 +210,15 @@ contract EmbalmerFacet {
             if (
                 sarcophagus
                     .cursedArchaeologists[selectedArchaeologists[i].archAddress]
-                    .doubleHashedKeyShare != 0
+                    .publicKey
+                    .length != 0
             ) {
-                revert LibErrors.ArchaeologistListNotUnique(selectedArchaeologists[i].archAddress);
+                revert ArchaeologistListContainsDuplicate(selectedArchaeologists[i].archAddress);
             }
 
-            // Validate the archaeologist has signed off on the sarcophagus parameters
-            LibUtils.verifyArchaeologistSignature(
-                selectedArchaeologists[i].doubleHashedKeyShare,
-                arweaveTxIds[1],
-                sarcophagusParams.maximumRewrapInterval,
-                sarcophagusParams.creationTime,
-                selectedArchaeologists[i].diggingFee,
-                selectedArchaeologists[i].v,
-                selectedArchaeologists[i].r,
-                selectedArchaeologists[i].s,
-                selectedArchaeologists[i].archAddress
-            );
+            // todo: check convenience structure for public keys that have already been used
+
+            // todo: verify that the sarcophagus parameters have been signed with the private key corresponding to the supplied public key
```

Comment:
Is this going to be extra gas costs to check the specific private key, rather than the arch's registering priv key (b/c we already have address for that key)? If it's not any more expensive, then can go that route, but if it is, then should discuss the tradeoff.

Checking the specific private key => pub key relationship ensures the archaeologist has access to that private key, but the archaeologist has no incentive *not* to provide a public key they don't have the private key for (if they did this, they wouldn't be able to unwrap, losing them $$$ and hurting their reputation).